renderer/index.js

```diff
@@ -216,6 +216,16 @@ function dispatch (action, ...args) {
   if (action === 'setDimensions') {
     setDimensions(args[0] /* dimensions */)
   }
+  if (action === 'backToList') {
+    while (state.location.hasBack()) state.location.back()
+
+    // Work around virtual-dom issue: it doesn't expose its redraw function,
+    // and only redraws on requestAnimationFrame(). That means when the user
+    // closes the window (hide window / minimize to tray) and we want to pause
+    // the video, we update the vdom but it keeps playing until you reopen!
+    var mediaTag = document.querySelector('video,audio')
+    if (mediaTag) mediaTag.pause()
```

Comment:
I wonder if we still need to pause if we're going back? Won't the video tag be unloaded? Or will that still not happen because it won't happen until `requestAnimationFrame`?